src/main/java/nextstep/courses/domain/Size.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.courses.domain;
+
+public class Size {
+    private static final long MAX_SIZE = 1_048_576L;
```

Comment:
관례적으로 용량은 바이트 단위 수식으로 나타내면 어떨까요?
```suggestion
    private static final long MAX_SIZE = 1 * 1024 * 1024;
```